app/code/pagetype/ContentPage.php

```diff
@@ -3,8 +3,44 @@
 namespace App\PageType;
```

Comment:
I see, so `ContentPage` in SilverStripe represents a generic page which means we can use `Page.php` instead of creating a new page type for `ContentPage`. 

We want to keep in mind that `Page.php` will be extended by multiple page types hence if particular fields are not used in other page types then you should remove those fields from the CMS.

Reply:
For further clarity we want to get rid of `ContentPage` and use `Page` instead :)